wazp/pages/02_ROI.py

```diff
@@ -137,42 +139,46 @@
 )
 
 # Buttons for saving/loading ROIs
-roi_save_button = dbc.Button(
-    "Save ROIs",
+disabled_button_style = {
+    "n_clicks": 0,
+    "outline": False,
+    "color": "dark",
+    "disabled": True,
+    "class_name": "w-100",
+}
+
+save_rois_button = dbc.Button(
+    "Save all to file",
     id="save-rois-button",
     download="rois.yaml",
-    n_clicks=0,
-    outline=False,
-    color="dark",
-    active=True,
+    **disabled_button_style,
 )
-roi_load_button = dbc.Button(
-    "Load ROIs",
-    id="load-rois-button",
-    n_clicks=0,
-    outline=False,
-    color="dark",
-    active=True,
+load_rois_button = dbc.Button(
+    "Load all from file", id="load-rois-button", **disabled_button_style
+)
+delete_rois_button = dbc.Button(
+    "Delete selected", id="delete-rois-button", **disabled_button_style
 )
 infer_rois_button = dbc.Button(
-    "Infer ROIs",
-    id="infer-rois-button",
-    n_clicks=0,
-    outline=True,
-    color="dark",
-    active=False,
+    "Infer positions", id="infer-rois-button", **disabled_button_style
 )
 # Tooltips for ROI buttons
 save_rois_tooltip = dbc.Tooltip(
-    "Save the ROIs to the video's " ".metadata.yaml file",
+    "Save all ROIs to the video's .metadata.yaml file. "
+    "This will overwrite any existing ROIs in the file!",
     target="save-rois-button",
 )
 load_rois_tooltip = dbc.Tooltip(
-    "Load ROIs from the video's metadata.yaml file",
+    "Load all ROIs from the video's metadata.yaml file. "
+    "This will overwrite any existing ROIs in the app!",
     target="load-rois-button",
 )
+delete_rois_tooltip = dbc.Tooltip(
+    "Delete selected ROIs", target="delete-rois-button"
+)
 infer_rois_tooltip = dbc.Tooltip(
```

Comment:
since we have not really started with this one, I'd suggest we remove this button from v0.

Reply:
Sure, can do